### ISSUE TITLE
NullPointerException in VirtualMachineCustomImagesImpl

### ISSUE THREAD

**Issue by c-n-d** [Azure/azure-sdk-for-java#1797]
Hi,

NPE occurs when looking up a non existent Custom Image. Seems similar to Azure/azure-sdk-for-java#1603.

* Stack trace
```
java.lang.NullPointerException
	at com.microsoft.azure.management.compute.implementation.VirtualMachineCustomImagesImpl.wrapModel(VirtualMachineCustomImagesImpl.java:37)
	at com.microsoft.azure.management.compute.implementation.VirtualMachineCustomImagesImpl.wrapModel(VirtualMachineCustomImagesImpl.java:17)
	at com.microsoft.azure.management.resources.fluentcore.arm.collection.implementation.GroupableResourcesImpl$1.call(GroupableResourcesImpl.java:117)
	at com.microsoft.azure.management.resources.fluentcore.arm.collection.implementation.GroupableResourcesImpl$1.call(GroupableResourcesImpl.java:114)
        ...
	at com.microsoft.azure.management.resources.fluentcore.arm.collection.implementation.GroupableResourcesImpl.getByResourceGroup(GroupableResourcesImpl.java:109)
	at com.microsoft.azure.management.resources.fluentcore.arm.collection.implementation.GroupableResourcesImpl.getByResourceGroup(GroupableResourcesImpl.java:33)
```

* `pom.xml`
```pom
<dependency>
    <groupId>com.microsoft.azure</groupId>
    <artifactId>azure</artifactId>
    <version>1.1.0</version>
</dependency>
```

* Code sample

```Java
VirtualMachineCustomImage virtualMachineCustomImage =
    azure.virtualMachineCustomImages()
         .getByResourceGroup(resourceGroupName, nonExistentImageName);
```

Thanks

**Comment by anuchandy** [comment]
This is fixed

**Comment by anuchandy** [comment]
Fix is available as of v1.2 hence closing this.
